src/main/java/task/Main.java

```diff
@@ -0,0 +1,63 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Main {
+
+    private static final String TEMPLATE = "Name: %s, salary: %s, balance: %s";
+
+    public static void main(String[] args) {
+        List<Person> people = DataGenerator.generatePeople();
+        List<Person> peopleWithSuspiciousBankAccounts = new ArrayList<>();
+
+        people.stream()
```

Comment:
why all this logic is in main method? It is really hard to read. Should split it to several private methods

Reply:
Fixed